### PR TITLE
docs[DST-189]: write utils documentation

### PR DESCRIPTION
**Pull request by sarahgm**

### CODE REVIEW

File: docs/content/hooks-and-utils/utils/utils.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
```
Comment (sebald) [inline docs/content/hooks-and-utils/utils/utils.mdx:1]: Move to own pages?

File: docs/content/hooks-and-utils/utils/utils.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  )}
+/>
+```
+
+In this code you can see that `p-3` will be overwritten by `p-4`, so keep in mind the order of your class names.
+
+## cva function
+
+To write our styles for a component we use [cva](https://cva.style/docs). This library comes with a great utility function which we use to write styles for our components.
+If you want to style your own component you can use this function which we updated with some additional style checks.
```
Comment (sebald) [inline docs/content/hooks-and-utils/utils/utils.mdx:41]: add note that it will resolve conflicting tailwind classnames

**Review by sebald — CHANGES_REQUESTED** [review]

File: docs/content/hooks-and-utils/utils/utils.mdx (hunk 1 above)
Comment (sarahgm) [reply docs/content/hooks-and-utils/utils/utils.mdx:1]: done

File: docs/content/hooks-and-utils/utils/utils.mdx (hunk 2 above)
Comment (sarahgm) [reply docs/content/hooks-and-utils/utils/utils.mdx]: done

**Comment by sarahgm** [comment]
@sebald also added badges

**Review by sebald — APPROVED** [review]

**Pull request merged**